.gitignore

```diff
@@ -58,3 +58,4 @@ errorScreenshots
 generated-themes
 package-lock.json
 lib
+!/packages/webpack-config-terra/lib/
```

Comment:
I think it'll get confusing that some lib contents are compiled and some are not tbh

Reply:
I'm not worried, it's common convention that your code lives in lib if it doesn't need to be transpiled and if we did enable babel for this repo it would just copy contents from src to lib.